client/src/components/Header/Header.js

```diff
@@ -2,9 +2,8 @@ import styled from "styled-components";
 import "./Header.css";
 import React, { useState } from "react";
 import { Link, useHistory } from "react-router-dom";
-import { Button, Drawer, List } from "@material-ui/core";
```

Comment:
I removed the unused List import for now, but I do want to use it (or something similar) to make the menu look nicer in the future